plans/academic_posse_improvement.markdown

```diff
@@ -0,0 +1,105 @@
+We need to refine and enhance the posse structure to support mentors in being effective partners with Turing students.
+
+# Critical Challenges Planning
+
+## The Challenge
+
+* What Critical Challenge are you going to work on?
+
+We need to refine and enhance the posse structure to support mentors in being effective partners with Turing students.
+
+* Why is this a “challenge”?
+
+Posses can "fall apart".
+
+* Can it be solved? What would be value to the organization/team in solving it or working towards a solution?
+
+I don't think it can be 100 percent solved, the people will always be that "X Factor"
+
+* What happens if we fail to solve it / improve it?
+
+Things will continue as they have, with some people taking advantage of their posse, and some not.
+
+* What skills do you have that make you the right person to work on this challenge?
+
+I didn't feel my posse was that effective, and would have liked it to have been more so.
+
+
+## The Past
+
+* What’s been done so far?
+
+There's a new posse structure, and now we have posse points.
+
+* Was it effective? How do you know?
+
+So far it has been more effective than it has in the past. Posse points is competitive and for some people this brings them together.
+
+* Was this someone’s challenge before you? Why did they stop doing it or how did we make progress without addressing it?
+
+Rachel has been working on this, and continues to do so.
+
+
+## The Present
+
+* What are the first steps that you’re ready to execute now?
+
+I've been thinking about the possibility of rotating mentors after each module, to expose more people to the posses.
```

Comment:
Hmm. Interesting. This could get tricky (like what if one side of the mentor-mentee pair wants to continue working together, and the other wants out?). I wonder if it's worth trying for one module though.